docs/framework/20categories/20architecture/liquidity-networks.md

```diff
@@ -1,3 +1,24 @@
 ### Liquidity Networks
+Liquidity networks enable users to transfer already minted assets from one chain to another. With such protocols, there's no ‘bridging’ of assets, i.e., no assets move across chains. Instead, assets are exchanged between one user (liquidity providers, aka LPs) and another bridge user on different networks, resulting in a cross-chain swap. For example, a user receives the desired asset on the destination network via the LP. In return, the LP receives the user’s assets on the source network and fees for providing the desired asset. 
 
-Note: include HTLCs as a possible design approach
\ No newline at end of file
+Typically, LP funds within a liquidity network are collected and locked in a smart contract (liquidity pool) to facilitate quick cross-chain swaps. When a user initiates a transaction, the desired assets are released from the smart contract on the destination network. In essence, liquidity networks work similarly to peer-to-peer networks where LPs, acting as routers, own and distribute liquidity across chains. 
+
+The relationship between the LP and the users depends on the mechanism used by the liquidity network:
+
+**Atomic swaps using HTLCs (locally verified)** 
+
+Atomic swaps are cross-chain peer-to-peer swaps leveraging a technology called hash Timelock Contracts (HTLCs). HTLCs ensure that the cross-chain swaps are time-bound and are either executed within a given timeframe or don’t go through at all (and users’ funds are returned after a specific period). For example, [in Connext NXTP](https://github.com/connext/nxtp), users either receive the desired assets on the destination network instantly, or if the transaction fails, they can claim their assets on the source network after the locking period of 72h.
```

Comment:
H in hash should be capitalised.

Reply:
"at all (and users’ funds are returned after a specific period)"
I think should be:
"at all. The user's funds are returned after a specific period."

Reply:
"users either receive the desired assets"
I think will be clearer if it is
"the user either receives the desired assets"

The change indicates that there is only one user, one party, and not multiple, engaged with the LP.

Reply:
"desired assets on the destination network instantly"
I don't think this really is "instantly". Some readers might take this word literally. Maybe say something like "within a few blocks" or something like that.